tests/conftest.py

```diff
@@ -0,0 +1,172 @@
+import logging
+import os
+import pathlib
+from contextlib import nullcontext
+from pathlib import Path
+
+import datajoint as dj
+import pytest
+from element_interface.utils import QuietStdOut, find_full_path, value_to_bool
+
+from workflow_zstack.paths import get_volume_root_data_dir
+
+# ------------------- SOME CONSTANTS -------------------
+
+
+logger = logging.getLogger("datajoint")
+
+pathlib.Path("../example_data").mkdir(exist_ok=True)
+
+sessions_dirs = [
+    "subject1",
+]
+
+
+def pytest_addoption(parser):
+    """
+    Permit constants when calling pytest at command line e.g., pytest --dj-verbose False
+
+    Arguments:
+        --dj-verbose (bool):  Default True. Pass print statements from Elements.
+        --dj-teardown (bool): Default True. Delete pipeline on close.
+        --dj-datadir (str):  Default ./tests/user_data. Relative path of test CSV data.
+    """
+    parser.addoption(
+        "--dj-verbose",
+        action="store",
+        default="True",
+        help="Verbose for dj items: True or False",
+        choices=("True", "False"),
+    )
+    parser.addoption(
+        "--dj-teardown",
+        action="store",
+        default="True",
+        help="Verbose for dj items: True or False",
+        choices=("True", "False"),
+    )
+    parser.addoption(
+        "--dj-datadir",
+        action="store",
+        default="./tests/user_data",
+        help="Relative path for saving tests data",
+    )
```

Comment:
```suggestion
```

Reply:
I would suggest removing this function for simplicity.  Tear down can be handled by bringing down the MySQL container or the entire Dev Container.

Reply:
That's true. We don't need this anymore